### ISSUE TITLE
Dockerfile

### ISSUE THREAD

**Issue by banana-cakes** [tjcim/xxefr#2]
Can you create a dockerfile?

**Comment by tjcim** [comment]
Yes. I should have already done it. I am working on a cert at the moment that takes up all of my free time, but should be able to do this within the next two months or so.
